src/clientversion.cpp

```diff
@@ -116,17 +116,46 @@ std::string FormatSubVersion(const std::string &name, int nClientVersion, const
     if (!subverOverride.empty())
         return subverOverride;
 
+    // sanitize comments per BIP014, format user agent and check total size
+    std::vector<std::string> uacomments = {};
+    for (std::string &cmt : mapMultiArgs["-uacomment"])
+    {
+        if (cmt != SanitizeString(cmt, SAFE_CHARS_UA_COMMENT))
+            break;
+        uacomments.push_back(SanitizeString(cmt, SAFE_CHARS_UA_COMMENT));
+    }
+    // If this is a 32bit build then append an identifier since we'd like to know
+    // how many still run this configuration.
+    {
+        int temp = 0;
+        int *ptemp = &temp;
+        if (sizeof(ptemp) == 4)
```

Comment:
we care about 32bit only, but since that they are complementary we can infer 64bit by taking the total number of nodes and subtracting the 32bit nodes count

Reply:
on second thought I think Id add and `else` clause to show 64bit, so that we don't make unit tests outcome archs dependent.